backend/Dockerfile

```diff
@@ -28,4 +28,5 @@ RUN --mount=type=cache,target=/root/.nuget/packages dotnet publish /p:Informatio
 FROM base AS final
 WORKDIR /app
 COPY --from=publish /app/publish .
+USER 33:33
```

Comment:
I would have used `www-data:www-data` here since it's more readable.  If it turns out that it must be 33:33 due to the shared volume, I would suggest we add a comment saying that we are referring to the `www-data` user.